pkg/types/autok3s.go

```diff
@@ -39,7 +39,8 @@ type Metadata struct {
 	InstallScript   string      `json:"k3s-install-script,omitempty" yaml:"k3s-install-script,omitempty"`
 	Mirror          string      `json:"k3s-install-mirror,omitempty" yaml:"k3s-install-mirror,omitempty"`
 	DockerMirror    string      `json:"dockerMirror,omitempty" yaml:"dockerMirror,omitempty"`
-	DockerScript    string      `json:"dockerScript,omitempty" yaml:"dockerScript,omitempty"`
+	DockerArg       string      `json:"docker-arg,omitempty" yaml:"docker-arg,omitempty"`
```

Comment:
How to set multiple parameters?
Maybe there's no more than one arg for docker installation, would it be property to use `StringArray` instead of `string`?

Reply:
For example

install_docker.sh
```
P3=${P1}/${P2}
```

```sh
--docker-arg 'P1="xxxx" P2="xxxx"'
```

Reply:
Please add the example description to dockerArg CLI usage.

Reply:
Add more description for dockerArg  , please check.